lib/fakerbot/renderer.rb

```diff
@@ -36,9 +36,10 @@ def call
     attr_reader :crayon, :hash, :options, :output, :pager, :screen, :tree
 
     def build_tree
-      hash.reduce({}) do |h, (faker, methods)|
-        h.merge! node(faker, methods)
+      data_tree = hash.reduce({}) do |h, (faker, methods)|
+        h.merge! node(faker, methods&.sort)
```

Comment:
Nice touch on safe navigation. 👍

Reply:
thank you! :+1: specs wouldn't pass without it

Reply:
Yup, covers for the case when `methods` might be nil. I've also dropped in #11 . Good chance to increase the test coverage.

Reply:
good work :+1: :sparkles: :smile: